BootloaderCorePkg/Tools/SingleSign.py

```diff
@@ -63,7 +63,11 @@
 def get_openssl_path ():
     if os.name == 'nt':
         if 'OPENSSL_PATH' not in os.environ:
-            os.environ['OPENSSL_PATH'] = "C:\\Openssl\\"
+            openssl_dir = "C:\\Openssl\\bin\\"
+            if os.path.exists (openssl_dir):
+                os.environ['OPENSSL_PATH'] = openssl_dir
+            else:
+                os.environ['OPENSSL_PATH'] = "C:\\Openssl\\"
         if 'OPENSSL_CONF' not in os.environ:
             openssl_cfg = "C:\\Openssl\\openssl.cfg"

```

Comment:
is this still correct for CFG file ?

Reply:
With latest openssl installer, default CFG file is installed at C:\Program Files\Common Files\SSL\
It looks with openssl_conf is not mandatory for Slimboot operations.

Updated PR to set openssl conf with older versions only.